capsules/object_speed/capsule.py

```diff
@@ -0,0 +1,32 @@
+from vcap import (
+    BaseCapsule,
+    NodeDescription,
+    DeviceMapper,
+    IntOption
+)
+
+from .backend import Backend
+from .stream_state import StreamState
+
+
+class Capsule(BaseCapsule):
+    name = "object_speed"
```

Comment:
Kind of weird to see a capsule without a type (`detector`, `classifier`, etc) although I don't know what the type would be in this case. Would it be worth designating a type for miscellaneous stuff like this? Maybe `utility` or just `misc`?

Reply:
Yeah, I was a bit at a loss there. For now I decided to just not have the prefix, since it's a bit of a one-off. I put `object` since that's what it operates on :P

`reader`? `measurer`?

Reply:
I guess we could use something like `measurer` on capsules that don't do analytics themselves but instead look at the results of analytics and draw conclusions, although then you could argue that trackers should fall under this category as well. Maybe we should hold off on giving this thing a type for now, and see if a category emerges as we make more capsules in the future.

Reply:
`calculator`?

Reply:
I like `calculator`. Let's use that, and for other capsules that are basically just adding convenience attributes based on other inference.

Reply:
Done